src/blosc2/schunk.py

```diff
@@ -876,6 +1110,26 @@ def remove_postfilter(self, func_name, _new_ctx=True):
         -------
         out: None
 
+        Examples
+        --------
+        >>> import blosc2
+        >>> import numpy as np
+        >>> dtype = np.dtype(np.int32)
+        >>> chunk_size = 20_000 * input_dtype.itemsize
+        >>> storage = {"cparams": {"typesize": input_dtype.itemsize}, "dparams": {"nthreads": 1}}
+        >>> data = np.arange(500, dtype=np.int32)
+        >>> schunk = blosc2.SChunk(chunksize=chunk_size, data=data, **storage)
+        >>> # Define the postfilter function
+        >>> @schunk.postfilter(input_dtype)
+        >>> def postfilter(input, output, offset):
+        >>>     output[:] = input + offset + np.arange(input.size)
+        >>> out = np.empty(data.size, dtype=dtype)
+        >>> schunk.get_slice(out=out)
+        >>> print("Data slice with postfilter applied (first 8 elements):", out[:8])
+        Data slice with postfilter applied (first 8 elements): [ 0  2  4  6  8 10 12 14]
+        >>> schunk.remove_postfilter('postfilter')
+        >>> print("Original data (first 8 elements):", data[:8])
+        Original data (first 8 elements): [0 1 2 3 4 5 6 7]
```

Comment:
Since you removed the post filter, you can check that the original data is the same as what schunk.get_slice would return.